apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx

```diff
@@ -0,0 +1,871 @@
+                description: "Title of a quotes section",
+              })}
+            </Heading>
+            <blockquote>
+              <div data-h2-text-align="base(left)">
+                <OpenQuote
+                  data-h2-display="base(inline-block)"
+                  data-h2-width="base(x2) p-tablet(x3)"
+                  data-h2-margin="base(0 0 x.5 0) desktop(0 0 -x3 -x4)"
+                />
+              </div>
+              <p
+                data-h2-padding-bottom="base(x1)"
+                data-h2-font-weight="base(bold)"
+                data-h2-font-size="base(h2)"
+              >
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Having had the privilege of working closely with the Indigenous Apprentices and witnessing the immense talent and potential they possess, our IRCC team is confident that this investment will bring tremendous value to both our department and our apprentices themselves.",
+                  id: "YSnedz",
+                  description:
+                    "Quote from Darcy Pierlot about working with apprentices",
+                })}
+              </p>
+              <div data-h2-text-align="base(right)">
+                <CloseQuote
+                  data-h2-display="base(inline-block)"
+                  data-h2-width="base(x2) p-tablet(x3)"
+                  data-h2-margin="base(0 0 x.5 0) desktop(0 -x4 -x1.25 0)"
+                />
+              </div>
+              <cite data-h2-font-weight="base(bold)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "– Darcy Pierlot, Chief Information Officer and Assistant Deputy Minister, Immigration, Refugees and Citizenship Canada",
+                  id: "T4xKEw",
+                  description:
+                    "Quote attribution for Quote from Darcy Pierlot about working with apprentices",
+                })}
+              </cite>
+            </blockquote>
+          </div>
+        </div>
+      </div>
+
+      {/* ready for IT section */}
+      <div>
+        <div
+          data-h2-container="base(center, iap-home, x1) l-tablet(center, iap-home, x2)"
+          data-h2-position="base(relative)"
+        >
+          <div data-h2-padding="base(x5, 0, x2, 0)">
+            <div data-h2-flex-grid="base(stretch, x3, x1) p-tablet(stretch, x2, x1) l-tablet(stretch, x3, x1)">
+              <div
+                data-h2-flex-item="base(1of1) p-tablet(1of2)"
+                data-h2-order="p-tablet(1)"
+              >
+                <Heading
+                  data-h2-font-size="base(h3, 1)"
+                  data-h2-margin="base(0, 0, x1, 0)"
+                  data-h2-text-align="base(center) p-tablet(left)"
+                  data-h2-color="base(primary.darker)"
+                >
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "Indigenous talent ready for IT apprenticeships",
```

Comment:
Little less spacing above for mobile view

Current

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/5d12426d-f8e5-4eb2-bf79-1f9ad06cb309)

Figma

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/7a64f6aa-ffaf-4996-a954-f432bc5bf8f0)